If >=2, detect violation of One-Definition-Rule (ODR);
If ==1, detect ODR-violation only if the two variables "have different sizes");
We modify the level from 2 to 1.
--- a/llvm/projects/compiler-rt/lib/asan/asan_flags.inc
+++ b/llvm/projects/compiler-rt/lib/asan/asan_flags.inc
@@ -136,7 +136,7 @@ ASAN_FLAG(
     bool, detect_container_overflow, true,
     "If true, honor the container overflow  annotations. "
     "See https://code.google.com/p/address-sanitizer/wiki/ContainerOverflow")
-ASAN_FLAG(int, detect_odr_violation, 2,
+ASAN_FLAG(int, detect_odr_violation, 1,
           "If >=2, detect violation of One-Definition-Rule (ODR); "
           "If ==1, detect ODR-violation only if the two variables "
           "have different sizes")
